src/renderer/components/title-bar/title-bar.component.tsx

```diff
@@ -12,6 +12,51 @@ import { useService } from "renderer/hooks/use-service.hook";
 import { lastValueFrom } from "rxjs";
 import { useWindowControls } from "renderer/hooks/use-window-controls.hook";
 
+function TitleBarTags({
+    ipcService
+}: Readonly<{
+    ipcService: IpcService
+}>) {
+    const [previewVersion, setPreviewVersion] = useState("");
+    const [outdated, setOutdated] = useState(false);
+
+    useEffect(() => {
+        const requests: Promise<any>[] = [
+            lastValueFrom(ipcService.sendV2("current-version"))
+        ];
+        if (window.electron.platform === "linux") {
+            requests.push(lastValueFrom(ipcService.sendV2("check-update")));
+        }
+
+        Promise.all(requests).then(([ currentVersion, outdated ]) => {
+            handlePrerelease(currentVersion);
+            setOutdated(outdated);
+        });
+    }, []);
+
+    const handlePrerelease = (version: string) => {
+        if (version.toLowerCase().includes("alpha")) {
+            return setPreviewVersion("ALPHA");
+        }
+        if (version.toLowerCase().includes("beta")) {
+            return setPreviewVersion("BETA");
+        }
+    }
+
+    return <>
+        {previewVersion &&
+            <span className="bg-main-color-1 text-white dark:text-black dark:bg-white rounded-full ml-1 text-[10px] italic px-1 uppercase h-3.5 font-bold">
+                {previewVersion}
+            </span>
+        }
+        {outdated &&
+            <span className="bg-main-color-1 text-white dark:text-black dark:bg-white rounded-full ml-1 text-[10px] italic px-1 uppercase h-3.5 font-bold">
+                Outdated
```

Comment:
A missing translation?

Reply:
`Outdated` should do the work
For the color, try to use the `bg-warning-xxx` for the background color (adjust the text color if needed)